bedrock/firefox/templates/firefox/family/includes/macros.html

```diff
@@ -8,8 +8,10 @@
   title,
   anchor
 ) -%}
-<a class="c-module-tag" href="#{{ anchor }}">
+<a class="c-module-tag" href="#{{ anchor }}" tabindex="0">
```

Comment:
Added a `tabindex` so folks using keyboard navigation could copy the section URL

Reply:
👍 anchor elements should be natively tabbable so this attribute might be redundant - but I think there's a specific Firefox setting that needs to be checked before it tabs through links like other browsers

Reply:
Hmm possibly, it definitely wasn't tabbing for me